example/src/todoForm.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FormEvent, useContext, useEffect} from 'react';
+import {observer} from '../../src';
+import {TodoStoreContext} from './store';
+
+export const TodoForm = observer(() => {
+    const todoStore = useContext(TodoStoreContext);
+    const input = React.createRef<HTMLInputElement>();
+
+    useEffect(() => {
+        if (input.current) {
+            input.current.focus();
+        }
+    });
+
+    const addTodo = (evt: FormEvent) => {
```

Comment:
What about `useCallback` here?

Reply:
Good idea. I'm still working my way through all the new hooks included in React out of the box.